src/engine/bank01.asm

```diff
@@ -8142,12 +8142,167 @@ _TossCoin: ; 71ad (1:71ad)
 	ret
 ; 0x72ff
 
-	INCROM $72ff, $7354
+Func_72ff: ; 72ff (1:72ff)
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp $01
```

Comment:
When you see a well-known memory address (`wDuelType`) being read and then compared to a value (`$01`), then you should take a look around the projects for uses of `wDuelType` to find out what the value `$01` means. In this case, we already have a constant that you can use here.

Reply:
Replaced $01 with DUELTYPE_LINK